can_tools/scrapers/official/MD/md_vaccine.py

```diff
@@ -18,62 +16,32 @@ class MarylandCountyVaccines(ArcGIS, DatasetBase):
     source = "https://coronavirus.maryland.gov/#Vaccine"
     source_name = "Maryland Department of Health"
 
+    variables = {
+        "firstdose": variables.INITIATING_VACCINATIONS_ALL,
+        "percentfullyvaccinated": variables.PERCENTAGE_PEOPLE_COMPLETING_VACCINE,
+        "fullyvaccinated": variables.FULLY_VACCINATED_ALL,
+    }
+
     def fetch(self):
         return self.get_all_jsons("MD_COVID19_VaccinationByCounty", 0, "")
 
     def normalize(self, data):
         df = self.arcgis_jsons_to_df(data)
         df.columns = [x.lower() for x in list(df)]
-        df = df.rename(columns={"county": "location_name"}).query(
-            "location_name != 'Unknown'"
-        )
 
-        crename = {
-            "firstdose": CMU(
-                category="total_vaccine_initiated",
-                measurement="cumulative",
-                unit="people",
-            ),
-            "seconddose": CMU(
-                category="total_vaccine_completed",
-                measurement="cumulative",
-                unit="people",
-            ),
-            "percentfirstdose": CMU(
-                category="total_vaccine_initiated",
-                measurement="current",
-                unit="percentage",
-            ),
-            "percentseconddose": CMU(
-                category="total_vaccine_completed",
-                measurement="current",
-                unit="percentage",
-            ),
-        }
-
-        out = (
-            df.melt(id_vars=["location_name"], value_vars=crename.keys())
-            .assign(
-                dt=self._retrieve_dt("US/Eastern"), vintage=self._retrieve_vintage()
-            )
-            .dropna()
+        # singledose was added only after J&J vaccines started.
+        # If it appears it will only be reported in the fully vacinated
+        # column. We also wanted to include it in `firstdose` b/c we
+        # report "people with at least one dose"
+        if "singledose" in list(df):
+            df["firstdose"] += df["singledose"]
+
+        out = pipe(
```

Comment:
should this just be `out = self._rename_or_add_date_and_location(...)`?

Reply:
they are equivalent. If we are doing it this way (just one method, not in a chain), then your proposal makes sense to me.

I'll make a change